quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/model/parser/PropertiesParser.java

```diff
@@ -152,10 +152,6 @@ private boolean isStopReading(StopReading stop) {
 		case PropertyName:
 			return isColonSeparator() || isWhiteSpace();
 		case PropertyValue:
-			if ((current == ';' || current == '#') && isWhiteSpace(last)) {

```

Comment:
This is another change I made to the PropertiesParser. I made this change because, beforehand, when the parser read this text:
```
a = value # comment
```
the `PropertyValue` only had a value of `value`, and not `value # comment`.

This change makes it so that the `PropertyValue` now has a value of `value # comment`.

From what I understand, it seems correct to consider the `# comment` as part of the value. Because for example, in the [application configuration quickstart project](https://github.com/quarkusio/quarkus-quickstarts/tree/master/application-configuration), having `greeting.message = hello # hello` causes the `message` string to be `"hello # hello"`

Reply:
Excellent @xorye !